public_html/documentation.php

```diff
@@ -114,8 +123,30 @@ function build_sidebar_nav($elements) {
     $main_content .= $toc_nav;
     $main_content .= '</div></div>'; # end of the sidebar col
     # main content
-    $main_content .= '<div class="col-12 col-lg-8"><div class="rendered-markdown">' . $content . '</div></div>';
-
+    if (in_array($_GET['path'], ['docs/usage', 'docs/contributing'])) {
+        $main_content .= '<div class="col-12 col-lg-8"><div class="rendered-markdown">';
+        if ($_GET['path'] === 'docs/usage') {
+            $main_content .=
+                '<p>Here you can find all documentation on how to use nf-core pipelines and nf-core tools to start processing your data. For information about how to contribute to nf-core pipelines, modules and subworkflows, please see the <a href="docs/contributing">contributing</a> section.</p>';
```

Comment:
Instead of putting this content into the PHP file, can we have an `index.md` file in the directory roots? That would be much easier to find and edit.

Reply:
I'll look into it! 👍